### DISCUSSION TITLE
Play framework application is not running

### DISCUSSION THREAD

**Discussion by PH191116** [playframework/playframework#12225] [General]
Hi everybody! I am facing an issue with my application, every time I run it server starts running but when I try to send a HTTP request in this case a GET request. I am using scala version 2.11.12 on my project and I have the 3.3.1 version installed in my machine and I am using the 2.4.x version of Play framework and I have java 1.8 installed in my machine as well, however I get the following error: 
![Captura](https://github.com/playframework/playframework/assets/115385041/bb95110d-c5b9-4b16-a4e0-1b25eb17acdf)

**Comment by mkurz** [comment]
> I am using scala version 2.11.12 on my project and I have the 3.3.1 version installed in my machine

This has nothing to do with each other. If you have Scala 2.11.12 defined for you project, then this is what your project is using.

> I am using the 2.4.x version of Play framework 

2.4 is not supported anymore since loooong time. Please upgrade to the latest Play version, with the help of our migration guides:
* https://www.playframework.com/documentation/latest/Migration25
* https://www.playframework.com/documentation/latest/Migration26
* https://www.playframework.com/documentation/latest/Migration27
* https://www.playframework.com/documentation/latest/Migration28
* https://www.playframework.com/documentation/latest/Migration29
* https://www.playframework.com/documentation/latest/Migration30

Sorry, I can't help you with this error here.
Best,
Matthias
